intro-HPC/antwerpen/macros.tex

```diff
@@ -7,6 +7,8 @@
 \macro{\hpcname}{hopper}
 \macro{\loginnode}{login.hpc.uantwerpen.be}
 \macro{\loginhost}{ln02.hopper.uantwerpen.vsc}
+%TODO: RSA key fingerprint here
```

Comment:
@sbecuwe: please verify?
@JensTimmerman: open an issue to verify the fingerprints?